src/harness/harnessIO.ts

```diff
@@ -1082,6 +1082,13 @@ function getVaryByStarSettingValues(varyBy: string): ReadonlyMap<string, string
     const option = ts.forEach(ts.optionDeclarations, decl => ts.equateStringsCaseInsensitive(decl.name, varyBy) ? decl : undefined);
     if (option) {
         if (typeof option.type === "object") {
+            if (option.deprecatedKeys) {
+                const map = new Map(option.type.entries());
+                for (const key of option.deprecatedKeys) {

```

Comment:
Should we be warning or something here?

Reply:
This was leftover from me trying to figure out why I _added_ a new baseline. Basically we have this syntax in the harness I had never seen, `*,-es3`, which says "everything but es3". But since we need to still have es3 in the map to know how to parse it (and then error on it), it's there in the list.

Reply:
(then I forgot to unstage this change)

Reply:
Oh, I misunderstood when this code runs. Makes sense